composer.json

```diff
@@ -30,14 +30,14 @@
         "ext-json": "*",
         "murtukov/php-code-generator": "^0.1.4",
         "psr/log": "^1.0",
-        "symfony/config": "^4.3 || ^5.0",
-        "symfony/dependency-injection": "^4.3 || ^5.0",
-        "symfony/event-dispatcher": "^4.3 || ^5.0",
-        "symfony/expression-language": "^4.3 || ^5.0",
-        "symfony/framework-bundle": "^4.3 || ^5.0",
-        "symfony/options-resolver": "^4.3 || ^5.0",
-        "symfony/property-access": "^4.3 || ^5.0",
-        "webonyx/graphql-php": "^0.13.5"
+        "symfony/config": "4.4 - 5.1",
```

Comment:
this syntax limit from 4.4.* to 5.1.* vs old syntax that allowed 4.3 to 5.*. Moving from 4.3 to 4.4 is not an issue but limiting the bundle to 5.1 is not what we want here.

Reply:
5.1 is the last stable version of Symfony. I restricted versions which pass Travis tests. If we allow 5.* it can break apps of users, when a new version of symfony is released, because it will be automatically installed. It is dangerous to allow untested versions to be installed, isn't it?

Reply:
Symfony 5 will no have any BC until 6 this mean that the bundle will support it without any major changes.

Reply:
Should I change it to `4.3 - 5.*`?

Reply:
`4.4 - 5.*` is ok

Reply:
I checked, this syntax is invalid (`4.4 - 5.*`), so i just used your old syntax with 4.4 instead of 4.3